### PR TITLE
docs/aws-iam-policies: sync with prod, re-structure

### PR DESCRIPTION
**Pull request by dustymabe**
Restructure the docs here to link to the prod version in the
Fedora infra ansible repo and update the community account version
to match latest prod.

### CODE REVIEW

File: docs/aws-iam-policies/README.md [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+account are stored in the [infra ansible repo](https://pagure.io/fedora-infra/ansible).
 
 #### Prod Account:
 
 - `fcos-builds-bot` user has the following policies attached
-    - [fcos-upload-amis](prod-account/fcos-upload-amis)
-    - [fcos-poc-artifacts](prod-account/fcos-poc-artifacts)
+    - [fcos-upload-amis.json](https://pagure.io/fedora-infra/ansible/blob/main/f/files/aws/iam/policies/fcos-upload-amis.json)
+        - Allows uploading/creating AMIs
+    - [fcos-poc-artifacts.json](https://pagure.io/fedora-infra/ansible/blob/main/f/files/aws/iam/policies/fcos-poc-artifacts.json)
```
Comment (jlebon) [inline docs/aws-iam-policies/README.md:11]: Aside: is the `poc` here for "proof of concept"? Obviously not high-priority, but we should drop that at some point.

File: docs/aws-iam-policies/README.md [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
 
 #### Community Account
 
 - The `prod-account-match-fcos-builds-bot` group has the following policies attached:
-    - [prod-account-match-fcos-upload-amis](community-account/prod-account-match-fcos-upload-amis)
-    - [prod-account-match-fcos-poc-artifacts](community-account/prod-account-match-fcos-poc-artifacts)
+    - [prod-account-match-fcos-upload-amis](community-account/prod-account-match-fcos-upload-amis.json)
+        - Allows uploading/creating AMIs
+        - Compare to prod with
+            - `vimdiff https://pagure.io/fedora-infra/ansible/raw/main/f/files/aws/iam/policies/fcos-upload-amis.json community-account/prod-account-match-fcos-upload-amis.json`
```
Comment (jlebon) [inline docs/aws-iam-policies/README.md:20]: Neat! TIL vim supports automatically fetching remote URLs to a tempdir. And it correctly deletes it on exit. That's definitely useful.

**Review by jlebon — APPROVED** [review]

File: docs/aws-iam-policies/README.md (hunk 1 above)
Comment (dustymabe) [reply docs/aws-iam-policies/README.md:11]: yep. I had the same exact thought, but I decided to not do the "dusty thing" for once and leave it (would need to PR upstream).

File: docs/aws-iam-policies/README.md (hunk 2 above)
Comment (dustymabe) [reply docs/aws-iam-policies/README.md:20]: quite handy, right?

**Pull request merged**